content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary_YubiHSM_2_Device.adoc

```diff
@@ -101,130 +126,88 @@ One
 Enter wrap keyID (0 to choose automatically):
 ....
 
-*Step 6*: You are prompted to create a wrap key and enter its ID. Do one of the following:
+*Step 6*: You are prompted to generate a wrap key and enter its ID. Do one of the following:
 
 * To manually assign a wrap key ID, type a number for the ID. As object ID `1` is already in use by the default application authentication key, we recommend assigning ID `2` to the wrap key.
 * To allow the system to generate a wrap key ID automatically, type `0`.
 
-In both cases, a confirmation message is displayed:
+In both cases, a confirmation message like the following is displayed:
 
-`Stored wrap key with ID 0x0002 on the device`
+....
+Stored wrap key with ID 0x0002 on the device
+....
 
-*Step 7*: You are prompted to configure splitting the wrap key among a number of key custodians. For this example, we will assume that the wrap key is split into three shares, out of which at least two shares must be present in order to regenerate the key. If only one key custodian is present, the wrap key cannot be regenerated.
+*Step 7*: You are prompted to specify the number of shares into which the wrap key should be split in order to be distributed to an equal number of key custodians. You are also prompted to specify the privacy threshold, which is the number of shares that must be present for the wrap key to be regenerated. For this example, we  assume that the wrap key is split into three shares, of which at least two shares must be present in order to regenerate the key.
 
-_Note: Each key custodian must record his or her share and store it safely in order to be able to regenerate the wrap key for this YubiHSM 2 device in the future._
+[Note]
+======
+For an overview of key custodian activities, see "Key Splitting and Key Custodians" in link:Prerequisites_and_Preparations.adoc[Prerequisites and Preparations]._
```

Comment:
There's a dangling _ at the end here confusing the formatting

Reply:
Removed - thanks for reviewing and instructing on the corrections!